src/app/main/component/user/components/habit/add-new-habit/habit-invite-friends/habit-invite-friends-pop-up/habit-invite-friends-pop-up.component.ts

```diff
@@ -26,6 +26,10 @@ export class HabitInviteFriendsPopUpComponent implements OnInit, OnDestroy {
     private localStorageService: LocalStorageService
   ) {}
 
+  get isAnyFriendSelected(): boolean {
+    return this.friends && this.friends.some((friend) => friend.added);
```

Comment:
You can use an optional chain like this: ` return this.friends?.some(friend => friend.added) ?? false;`

Reply:
update